src/components/Image/index.tsx

```diff
@@ -1,11 +1,39 @@
-import React, {useMemo} from 'react';
+import React, {useCallback, useMemo, useState} from 'react';
 import {withOnyx} from 'react-native-onyx';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
 import BaseImage from './BaseImage';
-import type {ImageOnyxProps, ImageOwnProps, ImageProps} from './types';
+import type {ImageOnLoadEvent, ImageOnyxProps, ImageOwnProps, ImageProps} from './types';
 
-function Image({source: propsSource, isAuthTokenRequired = false, session, ...forwardedProps}: ImageProps) {
+function Image({source: propsSource, isAuthTokenRequired = false, session, onLoad, objectPositionTop, style, ...forwardedProps}: ImageProps) {
```

Comment:
What do you think about changing `objectPositionTop: boolean` to `objectPosition: 'top' | 'initial'`? The whole thing has been inspired by CSS from the beginning, so maybe we can make the one last step to model this similarly to how CSS does and also analogously to many inline styles.

`initial` would be the default and would trigger the default behavior. [I took the name from CSS](https://developer.mozilla.org/en-US/docs/Web/CSS/initial).

Sorry about not sugesting this earlier, this just came to my mind.

Reply:
Great, let's do it 👍